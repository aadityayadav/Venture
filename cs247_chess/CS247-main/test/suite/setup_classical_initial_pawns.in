setup
+ k c6
+ K f3
+ p a1
done
- a1
+ p h1
done
- h1
+ P a8
done
- a8
+ P h8
done
- h8
+ P a2
+ P a3
+ P a4
+ P a5
+ P a6
+ P a7
+ p h2
+ p h3
+ p h4
+ p h5
+ p h6
+ p h7
done
