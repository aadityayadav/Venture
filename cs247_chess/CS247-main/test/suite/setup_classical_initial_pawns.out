The initial board cannot have pawns on the first or last row!
The initial board cannot have pawns on the first or last row!
The initial board cannot have pawns on the first or last row!
The initial board cannot have pawns on the first or last row!
________
P______p
P_k____p
P______p
P______p
P____K_p
P______p
________
